haiku-apps/bamkeys/bamkeys-1.0.0.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="Spatially map your display to navigate windows with the keyboard"
+DESCRIPTION="BamKeys let's you break your display up into \
+a grid of zones. Each zone is assigned a keyboard combination. \
+
+Whenever you press that keyboard combination, BamKeys will focus \
+the front-most window that intersects with that zone, regardless \
+of the program it belongs to! Pressing the same zone's keyboard \
+combination repeatedly will focus widows further back. If you have \
+three windows that intersect a zone, pressing that zones hotkey three \
+times will cycle through all those windows. \
+
+Configuring - \
+Open up the BamKeys Application (it's a preference tool) At this point, \
```

Comment:
You could call it "BamKeys preferences" and remove what's now in brackets. Maybe:

> Open up the BamKeys preferences: you'll see a little display with a 3x3 grid...